examples/nidcpower/measure-record.py

```diff
@@ -166,12 +172,27 @@ def on_close(event):
             fetch_multiple_response = client.FetchMultiple(nidcpower_types.FetchMultipleRequest(
                 vi = vi,
                 channel_name = channels,
-                timeout = get_measure_record_delta_time.attribute_value,
+                timeout = 5,
```

Comment:
`timeout` specifies the maximum time allowed for this function to complete (in seconds). In case of real hardware, it takes greater time to complete the function as compared to what is provided by `get_measure_record_delta_time.attribute_value`. If we use `get_measure_record_delta_time.attribute_value` we get some error as shown below.
![image](https://user-images.githubusercontent.com/78592845/116699039-bf2c8100-a9e2-11eb-9671-bf3cb3a789b0.png)
It says to either considering to increase provided timeout value or check the trigger configuration. For this example there is no configuration being done to the trigger, hence I changed the timeout value to 5 with proper testing.